@navikt/core/react/src/slot/Slot.tsx

```diff
@@ -10,11 +10,16 @@ const Slot = React.forwardRef<HTMLElement, SlotProps>((props, forwardedRef) => {
   const { children, ...slotProps } = props;
 
   if (React.isValidElement(children)) {
+    const childRef = Object.prototype.propertyIsEnumerable.call(
+      children.props,
+      "ref",
+    )
```

Comment:
I tried doing `children.props.ref || (children as any).ref` first, but it didn't work properly.